src/main/resources/standardTestCases/refresh_token_failure.test

```diff
@@ -0,0 +1,14 @@
+name: refresh_token_failure
+testFormat: workflowfailure
+backends: [Jes-Refresh]
+
+files {
+  wdl: hello/hello.wdl
+  inputs: hello/hello.inputs
```

Comment:
Is there any way to somehow test that input reading is included in the refresh token auth?